influxql/parser.go

```diff
@@ -1508,6 +1508,8 @@ func (p *Parser) parseUnaryExpr() (Expr, error) {
 	case DURATION_VAL:
 		v, _ := ParseDuration(lit)
 		return &DurationLiteral{Val: v}, nil
+	case MUL:
+		return &Wildcard{}, nil
```

Comment:
Sounds like a reasonable suggestion, but outside the scope of this change I'd say. Happy to open an issue.